duet/sys/config.g

```diff
@@ -72,26 +70,19 @@ M106 P1 S0
 M106 P2 S0
 
 ; Tool definitions
-M563 P0 D0 H1                           ; Define tool 0
-G10 P0 S0 R0                            ; Set tool 0 operating and standby temperatures
-;*** If you have a single-nozzle build, leave the next 2 lines commented out.
-;M563 P1 D1 H2                          ; Define tool 1
-;G10 P1 S0 R0 X0 Y17                    ; Set tool 1 operating and standby temperatures
-
-; Z probe and compensation definition
+M563 P0 S"config-user not set" D0 H1    ; Define tool 0
+G10 P0 S-273 R-273                      ; Set tool 0 operating and standby temperatures
 
-;*** Any file commented out with semi-colons is deactivated.
-;*** Do not use the Fxxx parameter for M558 in these files, as it is set in homez.g
-;*** WARNING - **Always** make sure your cables are correct, secure and test them thoroughly before homing Z for the first times!
+; Z probe
+M558 H10 A1 T3000  S0.02 ; Z probe - raise probe height.
+						 ; H10 - dive height
+						 ; A bigger dive height prevents a situation where the bed is out of alignment by more than the dive height
+						 ; on any corner, which can crash the hot-end into the bed while moving the head in XY.
+						 ; Probing speed and travel speed are similarly reduced in case the Z probe isn't connected properly (or
+						 ; disconnects later after moving to a point) giving the user more time to stop.
 
-;M98 P"zprobe/ir.g"                      ; Run IR Probe configuration file.
-;M98 P"zprobe/switch.g"                  ; Run Switch Probe configuration file. (untested/community submitted)
-;M98 P"zprobe/bltouch.g"                 ; Run BLTouch configuration file.
-;M98 P"zprobe/orion.g"                   ; Run Piezo Orion configuration file (untested/community submitted)
+M557 X50:200 Y50:200 S150 S150          ; Set Default Mesh (conservative)
 
-; *** Conservative settings enabled during build and testing. When you are ready to remove this ***
-; *** simple protection, remove or comment out this line                                        ***
-M98 P"conservative.g"
+M98 P"config-user.g"                    ; Load custom user config
```

Comment:
Does order here matter? If someone sets a PID tuning here, wouldn't the M501 below override it? Not sure on the best order.

Reply:
Actually conservative.g should be gone - it's all performed by config.g now.
So good catch regardless

Reply:
erm, it's not there
config.g only goes to line 88 - are you looking at something else here?

Reply:
Yes, there isn't both. conservative.g has gone.
So the situation is
config.g -> "load all common stock values, and many with conservative settings"
config-user.g -> "custom values, including speeds,accel,jerk etc to override conservative settings"
I hoping that makes sense to people :)

Reply:
There is a very rough README at
https://github.com/railcore/configs/blob/config-user/duet/sys/config-user%20examples/README.md
maybe it should be in /sys instead

(yes it will be fleshed out)